cypress/e2e/brewery/ResultsDashboards-DisplayDisabled.cy.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import { Login, ScenarioParameters, Scenarios } from '../../commons/actions';
+import { GENERIC_SELECTORS } from '../../commons/constants/generic/IdConstants';
+import { stub } from '../../commons/services/stubbing';
+import { routeUtils as route } from '../../commons/utils';
+import {
+  WORKSPACE_LIST_WITHOUT_DASHBOARDS,
+  WORKSPACE_WITHOUT_DASHBOARDS,
+} from '../../fixtures/stubbing/ScenarioViewDashboard/workspace';
+import { DEFAULT_SCENARIOS_LIST } from '../../fixtures/stubbing/default';
+
+const { id: scenarioId } = DEFAULT_SCENARIOS_LIST[0];
+const runOptions = {
+  runDuration: 1000,
+  dataIngestionDuration: 1000,
+  finalStatus: 'Successful',
+  expectedPollsCount: 2,
+};
+
+describe('results display is disabled', () => {
+  before(() => {
+    stub.start();
+    stub.setWorkspaces(WORKSPACE_LIST_WITHOUT_DASHBOARDS);
+  });
+
+  beforeEach(() => {
+    Login.login({
+      url: `/${WORKSPACE_WITHOUT_DASHBOARDS.id}`,
+      workspaceId: WORKSPACE_WITHOUT_DASHBOARDS.id,
+      isPowerBiEnabled: false,
+    });
+  });
+
+  after(() => {
+    stub.stop();
+  });
+
+  it('can launch a scenario and show Display of results is disabled dashboard', () => {
+    cy.get(GENERIC_SELECTORS.dashboards.tabName).should('not.exist');
+    ScenarioParameters.launch({ scenarioId, runOptions, saveAndLaunch: true });
+    ScenarioParameters.waitForScenarioRunEnd();
+    Scenarios.getDashboardAccordionLogsDownloadButton().should('be.visible');
+    Scenarios.getDashboardAccordion().click();
+    Scenarios.getDashboardPlaceholder().should('be.visible');
+    Scenarios.getDashboardPlaceholder().should(
+      'have.text',
+      'Scenario run was successful but the display of results is disabled'
+    );
+  });
+  it('can redirect from dashboards to workspace view when results display is disabled', () => {
+    route.browse({
+      url: `${WORKSPACE_WITHOUT_DASHBOARDS.id}/dashboards`,
+      workspaceId: WORKSPACE_WITHOUT_DASHBOARDS.id,
+      expectedURL: `${WORKSPACE_WITHOUT_DASHBOARDS.id}/scenario/${scenarioId}`,
+      isPowerBiEnabled: false,
+    });
+    cy.get(GENERIC_SELECTORS.dashboards.tabName).should('not.exist');
```

Comment:
Could we add the opposite check with `should('exist')` in another dashboard-enabled test? By checking this it once in another test, we can be sure that the selector is correct (otherwise, `should('not.exist')` could be working just because of a typo in the selector, for instance :smile: )